Changelog fixes 2024-05-04 (#3875)
sandboxie-plus/Sandboxie: CHANGELOG.md

- fixed UGlobalHotkey library not being compatible with Qt6

### Removed

> - removed hardcoded support for LogAPI.dll

Reviewer: ```suggestion
- removed hardcoded support for LogAPI library
```

- improved RPC debugging
- improved IPC handling around RpcMgmtSetComTimeout; "RpcMgmtSetComTimeout=n" is now the default behaviour
  - required exceptions have been hard-coded for specific calling DLLs

> - the LogAPI.dll is now using Sbie's tracing facility to log events instead of its own pipe server

Reviewer: ```suggestion
- the LogAPI library is now using Sandboxie's tracing facility to log events instead of its own pipe server
```

### Added
- created a new Qt-based UI named SandMan (Sandboxie Manager)
- Resource Monitor now shows the PID

> - added basic API call log using updated BSA LogAPI.dll

Reviewer: ```suggestion
- added basic API call log using updated BSA LogAPI library
```
